package.json

```
@@ -64,32 +42,32 @@
     }
   },
   "scripts": {
-    "prepare": "husky install && (forge --version || yarn foundryup) && yarn workspaces run prepare",
+    "prepare": "husky install && (forge --version || foundryup)",
```

If `forge --version` fails, we probably have to assume `foundryup` is not installed yet and should run the package script to install it?
```suggestion
    "prepare": "husky install && (forge --version || pnpm foundryup)",
```